### ISSUE TITLE
The Composer doesn't have a focus indicator on the web app

### ISSUE THREAD

**Issue by mehm8128** [bluesky-social/social-app#10569]
### Steps to Reproduce

1, move to the Composer using keyboard Tab key on the web app
2. you can find that the Composer doesn't show the focus indicator

### Attachments

[chrome_yTniy9JxEL.mp4](https://github.com/user-attachments/assets/5aa60dac-5101-45a4-b26b-cea438c451af)

### What platform(s) does this occur on?

Web (Desktop)

### Device Info

Windows 11, Chrome 148

### What version of the app are you using?

Build version: 1.121.0; Bundle info: 361578e (prod); Bundle date: 26050511; Platform: web; Platform version: 0.0.0; Device ID: c76f4a7f-1635-4f3b-b7e2-9054e234f23e9

### Additional Information

The cause is that `outline: none` is provided to the Composer,
https://github.com/mehm8128/social-app/blob/1b546c54db2d8483fd065499f1a511e977280e06/src/view/com/feeds/ComposerPrompt.tsx#L165-L168

**Comment by Zaven477** [comment]
@mehm8128 
Hello! Am I correct in understanding that the error is that the post input field in the modal window doesn't receive focus when switching using Tab?

**Comment by mehm8128** [comment]
@Zaven477 
Hello! I'm sorry I wasn't able to tell the bug accurately.
I wanted to say that the modal window trigger button, which includes the user icon, the text "What's up?" and image icon (enclosed in a red square in the following screenshot) doesn't show the focus indicator even when it has focus.

<img width="757" height="200" alt="the user icon, the text 'What's up?' and image icon is enclosed in a red square" src="https://github.com/user-attachments/assets/0fb098a6-1d69-419d-bc77-6c23d2d94bdc" />

**Comment by mehm8128** [comment]
To compare, I expect such focus indicator (I removed `outline: none;` in the devtools).

<img width="771" height="209" alt="the modal window trigger button is enclosed in a black focus indicator" src="https://github.com/user-attachments/assets/c53e1c23-809f-414c-8cb8-d6e746eadc76" />

**Comment by mehm8128** [comment]
closed by ↑
